docs/ossm/ossm2-migration/README.md

```diff
@@ -6,7 +6,8 @@ This document details how to migrate from 2.6 to OpenShift Service Mesh 3.0.
 
 Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to 3.0, ensure you have done the following:
 
-- Upgrade your 2.6 OpenShift Service Mesh Operator to the latest release.
+- Run the [migration-checker script](migration-checker.sh) to detect any issues with your environment.
```

Comment:
Should this be the last step in the checklist?

Reply:
No strong preferences. Running the script after reading what should be done will help users to understand what the script is actually checking maybe.

Reply:
Moved it to the bottom